src/App.js

```diff
@@ -142,91 +173,78 @@ const tabBarIconMap = {
   Settings: 'icSettings',
 };
 
-/**
- * This workaround is similar to the one in TokenSelect.js
- *
- * It should be removed after we upgrade react-navigation to
- * to use the latest version of SafeAreaView
- */
-const tabNavigatorMarginWorkaround = () => {
-  const deviceId = DeviceInfo.getDeviceId();
-  const workaroundDeviceIds = [
-    'iPhone13,2', // iPhone 12
-    'iPhone13,3', // iPhone 12 Pro
-    'iPhone13,4', // iPhone 12 Pro Max
-    'iPhone14,5', // iPhone 13
-    'iPhone14,2', // iPhone 13 Pro
-    'iPhone14,3', // iPhone 13 Pro Max
-    'iPhone14,7', // iPhone 14
-    'iPhone14,8', // iPhone 14 Plus
-    'iPhone15,2', // iPhone 14 Pro
-    'iPhone15,3', // iPhone 14 Pro Max
-  ];
-
-  return workaroundDeviceIds.includes(deviceId);
+const TabNavigator = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)